packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -353,6 +491,8 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
     [emptyStateMessage],
   );
 
+  const [tagContainerWidthRef, tagContainerWidth] = useContainerWidth();
```

Comment:
This is used in Combobox input component inline styles. Should I move tagContainerWidth into context?

Reply:
yeah that's one option.

Alternatively, put `tagContainerRef` in context, and move `useContainerWidth` inside ComboBoxInput because that's the only place it is used